### PR TITLE
Add RPC example outputs - Closes #870

### LINKED ISSUE #870: Add missing example outputs to the RPC endpoints page
The RPC endpoints page has some missing example outputs for events and actions, that need to be added.

### Affected versions
v5

### PR DESCRIPTION
**Pull request by Tschakki**
Closes #870

### CODE REVIEW

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc [hunk 1]
```diff
@@ -450,29 +415,52 @@ Returns all connected peers.
...
+      "height": 3240,
+      "maxHeightPrevoted": 3143,
+      "blockVersion": 2,
+      "lastBlockID": "fc108ac8e2a39eb00ba0dc9bb2158f3e1b77b0c2dfd6f014ebcf4a5d6092f6de"
+    },
+    "ipAddress": "127.0.0.1",
+    "port": 5005,
+    "peerId": "127.0.0.1:5005"
+  }
+]
```
Comment (Incede) [inline modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:437]: Question: Would you prefer the output format like for `app:chain:validators:change` or as it is now ?

something like 
```
{ peers: [ { }, { } .... { } ] }
```

And maybe add an example output with multiple peers.

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc [hunk 2]
```diff
@@ -450,29 +415,52 @@ Returns all connected peers.
...
 ==== Input
 none
 
-////
 ==== Returns
 .Example output
 [source,json]
 ----
-[]
+[
```
Comment (Incede) [inline modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:450]: same, multiple peer output example if possible

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc [hunk 3]
```diff
@@ -521,19 +509,35 @@ Height of the previously last forged block.
...
 ==== Returns
 
-.Example output
+.Example output after disabling forging
 [source,json]
 ----
-
+{
+  "address": "d6a79b4f59f7da027405277b55783ba7dd255aa9"
+}
```
Comment (Incede) [inline modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: What is this address in the example conveying here ?

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc [hunk 4]
```diff
@@ -521,19 +509,35 @@ Height of the previously last forged block.
...
 ----
-////
+
 
 === `app:getAccount`
 Gets information about an account based on its address.
 
+==== Example
+
+[source,js]
```
Comment (Incede) [inline modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:529]: Are you adding source for only selected examples ?

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc [hunk 5]
```diff
@@ -567,6 +595,7 @@ Gets information about multiple accounts based on their address.
 [source,json]
 ----
 [
+  "0a14d6a79b4f59f7da027405277b55783ba7dd255aa912070880d6d2c881031a020800220208002a190a170a0c64656c65676174655f313031180120ac1628003000c23e0e0a0c48656c6c6f20576f726c6421",
```
Comment (Incede) [inline modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:598]: It does not say here if it Is input or output

**Review by Incede — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:529]: Can you explain more? I am not sure I understand what you mean

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 1 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:437]: I don't have a strong opinion here, either is fine for me I guess.

About multiple outputs: It is a little cumbersome to generate these, and I don't think it is adding new information, so I would prefer to keep it as is.

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 2 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:450]: See my reply above

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: This is the data that is returned when disabling forging via RPC action (see label of the snippet)

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 5 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:598]: Yes it does, in the label above the snippet.

Comment (Incede) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:598]: ah yes! It was truncated in Github in default view

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 3 above)
Comment (Incede) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: hmm yea ik but I am trying to understand what does it mean... How is it related to `updateForgingStatus` ?

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 4 above)
Comment (Incede) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:529]: I saw in some places the source code for some examples was removed (L169, L235) or not present in others.

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: it is an example of what it returns

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:529]: I wanted to include one example of decoding an account, which I think is missing in the docs atm.
Probably there should be a guide covering this, but for now I added a simple example here.

The other two examples were kind of old and covered elsewhere in the docs, so I removed the examples for them.

Comment (Incede) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:529]: I'd suggest using a consistent approach for providing source code with examples (either do for all or none) but in the end it's up to you.

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 3 above)
Comment (Incede) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: hmm idk if it's the right behavior but returning only an address as the output of `app:updateForgingStatus` doesn't make sense for me.

Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: Maybe you can verify it yourself, for me this was exactly what was returned when disabling forging.
If you don't like what it returns, I suggest to create a new issue for it on the lisk-sdk repo.
I agree personally, more info, and maybe a general more standardized format for RPC replies would be great.
Also for `app:ready` and `app:shutdown` it only returned `{}` / undefined when the event was fired.

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 4 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:529]: For now it is OK like this, but if I create the guide, I will consider remove it here for consistency 👍

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 3 above)
Comment (Incede) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: Thanks for the feedback. Created an [issue](https://github.com/LiskHQ/lisk-sdk/issues/6967). Would be nice if you could also specify details what you expect in terms of "more information and standardized format" in the issue itself.

**Review by Incede — APPROVED** [review]

File: modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc (hunk 3 above)
Comment (Tschakki) [reply modules/ROOT/pages/advanced-explanations/rpc-endpoints.adoc:520]: Thanks!

**Pull request merged**